Frontend/src/4_features/EditableArticleCard/ui/EditableArticleCardHeader/EditableArticleCardHeader.tsx

```
@@ -56,10 +73,12 @@ export const EditableArticleCardHeader = memo(
         } else {
           result = await dispatch(saveArticleData()).unwrap();
         }
-
-        if (result && result.id) {
-          navigate(getRouteArticleDetails(result.id));
+        if (result && result?.article?.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)